vnext/PropertySheets/Bundle.props

```diff
@@ -36,7 +36,8 @@
     <UseDevBundle Condition="'$(UseDevBundle)' == '' and '$(Configuration)' != 'Debug'">false</UseDevBundle>
 
     <!-- Extra arguments to pass to the bundler command -->
-    <BundlerExtraArgs Condition="'$(BundlerExtraArgs)' == ''"></BundlerExtraArgs>
+    <BundlerExtraArgs Condition="'$(BundlerExtraArgs)' == '' AND '$(UseHermes)' != 'true'"></BundlerExtraArgs>
+    <BundlerExtraArgs Condition="'$(BundlerExtraArgs)' == '' AND '$(UseHermes)' == 'true'">--minify false</BundlerExtraArgs>
```

Comment:
>minify [](start = 93, length = 6)

Is this intentional? Shouln't it be debug only?

Reply:
It's intentional. The reason is that Hermes will convert the JS bundle to bytecode, so minification doesn't affect the final output. Just wastes some cycles.

Not sure how much the optimization does, but we now do this for Android/iOS.